lib/index.js

```diff
@@ -57,9 +56,9 @@ function getStatsdApi ({ hostname, port, lynxOptions, transmit, prefix }) {
 }
 
 module.exports = function ({ client }) {
-  const opts = client.config('plugins.statsd')
+  const options = client.config('plugins.statsd')
```

Comment:
Nit: we have a several other internal repos that use this same syntax -- can we leave it as `opts`?

Reply:
Oh, okay! That makes sense. 🚢